shared/message/support/SaveConfiguration.proto

```diff
@@ -20,7 +20,18 @@ syntax = "proto3";
 
 package message.support;
 
+/**
+ * Contains config file to save, and the path.
+ */
 message SaveConfiguration {
-    string path   = 1;
+
+    /**
+     * Relative path of the config save.
```

Comment:
Relative to what?

Reply:
so like, it's relative to the root of the main `_data` directory of a build. For example, on my machine it's relative to `/var/lib/docker/volumes/<build_name>/_data/` 
(at least that's what I think, based on this message's one use in `WalkOptimiser.cpp` line 197)
I am not sure how to describe that though?

Reply:
it's an excellent point though, one that I didn't think of at all

Reply:
I believe it's relative to the build directory. Updated it to reflect that.